tests/GraphQLTest.php

```diff
@@ -230,6 +227,22 @@ public function testAddType()
         $type = GraphQL::type('CustomExample');
         $this->assertInstanceOf(\GraphQL\Type\Definition\ObjectType::class, $type);
     }
+
+    /**
+     * Test that the add type event is fired when fire_type_added_events is true
+     *
+     * @test
+     */
+    public function testAddTypeFiresEventWhenEventFiringIsEnabled()
+    {
+        config(['graphql.fire_type_added_events' => true]);
```

Comment:
I believe this test got it backwards and should set it to false and test the event is not fired.

Reply:
I think the reason I didn't here is that Laravel 5.1 has no assertion for an event not being fired, it wasn't added until 5.2. So I couldn't test the inverse while the package still supports Laravel 5.1.